### PR TITLE
Docker machine commands

### PR DESCRIPTION
**Pull request by tomaslin**
changed the name of the docker machine to be spinnakergcp

use docker-machine scp and docker-machine ssh where appropriate. ( this will simplify instructions for azure and aws ) 

@duftler ptal

### CODE REVIEW

File: experimental/docker-compose/README.md [hunk 1]
```diff
@@ -53,27 +53,25 @@ Create a project that will hold your Spinnaker install and keep track of the pro
...
 
 You should see an IP address returned and an instance running in GCP.
 
 ## Copy configuration files to the GCP instance
 
-The next step is to copy over the configuration files from our local machine to our instance.  We'll use the `gcloud` cli tool to do this. 
+The next step is to copy over the configuration files from our local machine to our instance.  
 
-```gcloud compute ssh --project spinnakergce --zone us-central1-a --command "mkdir /home/ubuntu/spinnakerconfig" ubuntu@goocker```
+```docker-machine scp -r ../../config spinnakergcp:/home/ubuntu/spinnakerconfig```
```
Comment (duftler) [inline experimental/docker-compose/README.md]: Replace `/home/ubuntu` with `~`.

File: experimental/docker-compose/README.md [hunk 2]
```diff
@@ -53,27 +53,25 @@ Create a project that will hold your Spinnaker install and keep track of the pro
...
 
-```gcloud compute copy-files --project spinnakergce --zone us-central1-a ../../config/* ubuntu@goocker:~/spinnakerconfig```
-
-```gcloud compute copy-files --project spinnakergce --zone us-central1-a compose.env ubuntu@goocker:~/spinnakerconfig```
+```docker-machine scp compose.env spinnakergcp:/home/ubuntu/spinnakerconfig```
```
Comment (duftler) [inline experimental/docker-compose/README.md]: Replace `/home/ubuntu` with `~`.

File: experimental/docker-compose/README.md [hunk 3]
```diff
@@ -53,27 +53,25 @@ Create a project that will hold your Spinnaker install and keep track of the pro
...
 
 Ssh into the box:
 
-```gcloud compute ssh --project spinnakergce --zone us-central1-a ubuntu@goocker```
+```docker-machine ssh spinnakergcp```
```
Comment (duftler) [inline experimental/docker-compose/README.md]: The block below needs to be:
```
sudo cp -r ~/spinnakerconfig/ /root/spinnakerconfig/
sudo su
chmod 666 /root/spinnakerconfig
chmod 444 /root/spinnakerconfig/*
```

**Comment by duftler** [comment]
LGTM after addressing comments.
Tested all the way through via copy/paste.

**Comment by tomaslin** [comment]
Added Azure instructions and tested end to end with both GCP and Azure

**Comment by duftler** [comment]
Tried it again after latest commits and it worked fine.

**Comment by duftler** [comment]
LGTM.

**Pull request merged**